tox.ini

```diff
@@ -1,15 +1,22 @@
 [tox]
-envlist = py27, py33, py34, py35, flake8
+envlist = {py27,py33,py34,py35}-{windows,linux,darwin},flake8
 skipsdist=True
 
 [testenv]
 usedevelop=True
 commands =
     py.test -v --cov=docker {posargs:tests/unit}
+platform =
+    windows: windows
```

Comment:
This should be 
 
```
{py27,py33,py34,py35}-windows: windows
```

and ditto for the other relevant lines.

If you're asking whether I feel bad already for giving you this advice: yes but I have no better advice :(